charts/kargo/values.yaml

```
@@ -291,6 +291,15 @@ api:
       #     clientID: <your client ID>
       #     clientSecret: "$CLIENT_SECRET"
       #     redirectURI: <http(s)>://<api.host>/dex/callback
+      ## Azure Example
+      # - id: microsoft
+      #   name: microsoft
+      #   type: Microsoft
+      #   config:
+      #     clientID: <your client ID>
+      #     clientSecret: "$ENTRA_CLIENT_SECRET"
```

Just for consistency with the other examples:

```suggestion
      #     clientSecret: "$CLIENT_SECRET"
```